lib/grpc/client/adapters/mint/stream_response_process.ex

```diff
@@ -0,0 +1,178 @@
+defmodule GRPC.Client.Adapters.Mint.StreamResponseProcess do
+  @moduledoc """
```

Comment:
Is this a public module that people should be using?

Reply:
Nope! This is an internal module. You think I should declare this inside another module?

Reply:
Just set moduledoc false and turn the docstring into comments!